tests/CMakeLists.txt

```
@@ -43,6 +43,7 @@ add_fdb_test(TEST_FILES BandwidthThrottle.txt IGNORE)
 add_fdb_test(TEST_FILES BigInsert.txt IGNORE)
 add_fdb_test(TEST_FILES BlobStore.txt IGNORE)
 add_fdb_test(TEST_FILES ConsistencyCheck.txt IGNORE)
+add_fdb_test(TEST_FILES DDMetricsExclude.txt)
```

```suggestion
add_fdb_test(TEST_FILES DDMetricsExclude.txt IGNORE)
```

You want this to be IGNORE, right?  Otherwise a full CTest run will try to run this locally as a simulation test still.

---

Yes, good catch. I'll change that.